components/Badge/badge.css

```diff
@@ -0,0 +1,71 @@
+.bdg-av{
+    display:flex;
+    flex-direction: row;
+}
+:root{
+    --online:rgb(49, 218, 49);
+    --offline:yellow;
+    --onCall:red;
+    --whislist:rgb(209, 86, 86);
+    --notification:rgb(52, 139, 221);
+    --badge:white;
+}
+.badge-on-avtar{
+    position: relative;
+    
+    margin: 1rem;
+}
+.avtar{
```

Comment:
Insted of defining a new class, the Avatar Class, defined above can be used here.